src/services/videoconference/index.js

```diff
@@ -481,6 +483,14 @@ class CreateVideoconferenceService extends VideoconferenceBaseService {
 					settings,
 					true,
 				);
+				if (scopeName === 'course') {
+					await app.service('/courses').get(scopeId, {
+						query: { $populate: ['userIds'], },
+					}).then((users) => {
```

Comment:
add teams here tooo

Reply:
We can use the scope routes here: `/[scopeName]/[:scopeId]/members` with scopeName=courses || teams